gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1104,20 +1104,25 @@ void gnc_file_ofx_import (GtkWindow *parent)
         ofx_set_security_cb (libofx_context, ofx_proc_security_cb, &info);
         /*ofx_set_status_cb(libofx_context, ofx_proc_status_cb, 0);*/
 
+        DEBUG("Opening selected file(s)");
+        for (iter = selected_filenames; iter; iter = iter->next)
+        {
+            selected_filename = iter->data;
 #ifdef G_OS_WIN32
-        conv_name = g_win32_locale_filename_from_utf8(selected_filename);
-        g_free(selected_filename);
-        selected_filename = conv_name;
+            selected_filename = g_win32_locale_filename_from_utf8 (selected_filename);
 #endif
```

Comment:
Change this to...
#ifdef G_OS_WIN32
            char *selected_filename = g_win32_locale_filename_from_utf8 (iter->data);
#else
            char *selected_filename = g_strdup (iter->data);
#endif
            DEBUG("Filename found: %s", selected_filename);